debian/control

```diff
@@ -0,0 +1,17 @@
+Source: qubes-fwupd
+Section: misc
+Priority: extra
+Maintainer: Norbert Kamiński (Asiderr) <norbert.kaminski@3mdeb.com>
+Build-Depends:
+    debhelper (>= 10~),
+    config-package-dev
+Standards-Version: 1.3.1
+Homepage: https://fwupd.org/
+
+Package: qubes-fwupd-vm-whonix
+Section: misc
+Architecture: amd64
+Depends: gpg
+Provides: ${diverted-files}
```

Comment:
I think you don't need `${diverted-files}` in Provides and Conflicts. Normally, it's only if you need to `displace` files: a dpkg way for backuping and replace existing file that a package is supposed to override.

Reply:
You've right. I've removed the unnecessary configurations in the last commit.